File: python/source/dexi/controllers/abstract_controller.py
# -*- coding: utf-8 -*-
from dexi.api_helper import APIHelper


class AbstractController:
    def __init__(self, x_cloud_scrape_account, x_cloud_scrape_access):
        self.api_helper = APIHelper(x_cloud_scrape_account, x_cloud_scrape_access)

File: dotnet/source/dexi/controllers/AbstractController.cs
using System;
using System.IO;
using System.Net;
using System.Security.Cryptography;
using System.Text;
using System.Web.Script.Serialization;

namespace Dexi {

    using System;
    using System.Runtime.Serialization;

    [Serial
    public partial class AbstractController {

        protected string accountId;
        protected string accessToken;
        protected ApiHelper apiHelper;

        public AbstractController(String accountId, String accessToken) {
            this.accountId = accountId;
            this.accessToken = accessToken;
            this.apiHelper = new ApiHelper(accountId, accessToken);
        }
    }
}
